sql/src/test/java/io/crate/protocols/postgres/types/PGTypesTest.java

```diff
@@ -65,6 +65,12 @@ public void testCrate2PGType() {
             instanceOf(VarCharType.class));
     }
 
+    @Test
+    public void test_undefined_type_can_stream_non_string_values() {
```

Comment:
lgtm

Note: Maybe we can use `BasePGTypeTest` to test `VarCharType` and move other related tests there as well.